vmm/src/lib.rs

```diff
@@ -818,10 +801,9 @@ impl Vmm {
     pub fn new(
         api_shared_info: Arc<RwLock<InstanceInfo>>,
```

Comment:
nit: s/api_shared_info/shared_info (or something that doesn't explicitly name `api`)